Content.Shared/Roles/StartingGearPrototype.cs

```diff
@@ -36,10 +37,23 @@ public class StartingGearPrototype : IPrototype
         [DataField("id", required: true)]
         public string ID { get; } = default!;
 
-        public string GetGear(Slots slot, HumanoidCharacterProfile? profile)
+        public IReadOnlyDictionary<string, string> UniformAlternates => _uniformAlternates;
+        [DataField("uniformalternates")]
+        private Dictionary<string, string> _uniformAlternates = new(0);
+
+        public string GetGear(Slots slot, HumanoidCharacterProfile? profile, CrewUniformPreference? crewUniformPreference = CrewUniformPreference.Default)
         {
             if (profile != null)
             {
+                if (crewUniformPreference != null && crewUniformPreference != CrewUniformPreference.Default)
+                {
+                    var preference = crewUniformPreference.ToString();
+                    if (slot == Slots.INNERCLOTHING && preference != null && _uniformAlternates.ContainsKey(preference))
+                    {
+                        return _uniformAlternates[preference];
+                    }
+                }
```

Comment:
When crew uniforms are a prototype you can get by that instead of having to do a bunch of string checks.